### DISCUSSION TITLE
Testing local changes with 'yarn link' results in build errors in project using linked vuetify

### DISCUSSION THREAD

**Discussion by sburdinski** [vuetifyjs/vuetify#18903] [Q&A]
I am currently trying to test some issues and hopefully fixes regarding a11y in vuetify.
So I wanted to setup an environment for changing vuetify and testing these changes in my application.

Easily set up the environment using the `master` branch, did my research, tests in the playground, compiling and `yarn link` for connecting the changed vuetify package and my application.

When compiling my application, this error shows up:

[vite]: Rollup failed to resolve import "vuetify/lib/../src/components/VApp/index.mjs" from "....."

I think the `../src` part of the import is wrong, but I could not find any reference to this.

Anyone else ran into that?

Regards
Stefan

**Comment by joelmandell** [comment]
Most probably because building on Windows machine? Perhaps build it using WSL?
